examples/acmetool/responses

```diff
@@ -0,0 +1,15 @@
+# https://hlandau.github.io/acme/userguide#response-files
+#
+# This is a example of a response file, used with --response-file.
+# It automatically answers prompts for unattended operation.
+# grep for UniqueID in the source code for prompt names.
+# Pass --response-file to all invocations, not just quickstart.
+# If you don't pass --response-file, it will be looked for at "(state-dir)/conf/responses".
+# You will typically want to use --response-file with --stdio or --batch.
+# For dialogs not requiring a response, but merely acknowledgement, specify true.
+# This file is YAML. Note that JSON is a subset of YAML.
+"acme-enter-email": "hostmaster@example.com"
```

Comment:
maybe we need to generate this file in the entrypoint.sh on each startup? With the rancher-catalog we can pass possible vars as env-vars so this should be no problem.